js/screens/Activity/ActivityContainer.js

```diff
@@ -1,11 +1,39 @@
 import React, { Component } from 'react';
 import Activity from './Activity';
-
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { Text } from 'react-native';
+import PropTypes from 'prop-types';
 export default class ActivityContainer extends Component {
   static navigationOptions = {
     title: 'Curriculum',
```

Comment:
Is this right? Activity screen's title is Curriculum?

Reply:
Yep, this is according to the mockup.